InternalDelay.py

```diff
@@ -65,59 +51,14 @@ class InternalDelay:
     >>> s = utils.tf([1, 0], 1)
     >>> G = 1/(1.25*(s + 1)*(s + 2)) * utils.mimotf([[s - 1, s * numpy.exp(-2*s)], [-6, s - 2]])
 
-    >>> G_id = utils.InternalDelay(G)
+    >>> G_id = utils.InternalDelay.from_utils_mimotf(G)
```

Comment:
`from_mimotf` allows us to avoid mentioning the name `utils` which is likely to change in the future.

Reply:
Should I also refactor `from_utils_tf`?